### PR TITLE
feat: actor_school_blacklist table

### PR DESCRIPTION
**Pull request by Ajay-Maury**
actor_school_blacklist table  created

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -644,6 +645,7 @@ model school_list {
   lat                                          Decimal?                                       @db.Decimal
   long                                         Decimal?                                       @db.Decimal
   geo_fence_enabled                            Boolean                                        @default(true)
+  actor_school_blacklist                       actor_school_blacklist[]
```
Comment (tushar5526) [inline prisma/schema.prisma:648]: why this ?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -10,6 +10,7 @@ datasource db {
 model actors {
   id                                       Int                                        @id @default(autoincrement())
   name                                     String                                     @unique @db.Char
+  actor_school_blacklist                   actor_school_blacklist[]
```
Comment (tushar5526) [inline prisma/schema.prisma:13]: why we have this ?

Comment (Ajay-Maury) [reply prisma/schema.prisma:13]: foreign key mappings

File: prisma/schema.prisma [hunk 3]
```diff
@@ -1945,6 +1947,19 @@ model school_results_fraud_reports {
   @@index([udise], map: "school_results_fraud_reports_udise_key")
 }
 
+model actor_school_blacklist {
+  id          Int         @id @default(autoincrement())
+  actor_id    Int
+  udise       BigInt
+  is_active   Boolean     @default(true)
+  created_at  DateTime    @default(now()) @db.Timestamptz(6)
+  updated_at  DateTime    @default(now()) @db.Timestamptz(6)
```
Comment (tushar5526) [inline prisma/schema.prisma:1956]: Updated at should be configure as follows - https://github.com/prisma/prisma/discussions/20323

File: src/admin/admin.controller.ts [hunk 4]
```diff
@@ -252,4 +253,40 @@ export class AdminController {
       throw new InternalServerErrorException({ error: 'Failed to upload zip file', message: error.message });
     }
   }
+
+  @Post('/school/blacklist')
+  @Roles(Role.Admin)
+  @UseGuards(JwtAdminGuard)
+  @Throttle({ default: { limit: 100, ttl: 60000 } })
+  async createSchoolActorBlacklist(
```
Comment (tushar5526) [inline src/admin/admin.controller.ts:261]: There should be a DELETE endpoint here as well.

Comment (tushar5526) [reply src/admin/admin.controller.ts:261]: Also add in the new endpoints in the NL APIs postman collection.

**Review by tushar5526 — CHANGES_REQUESTED** [review]
Please look at the comments.

**Review by tushar5526 — APPROVED** [review]

**Pull request merged**